src/main/kotlin/br/com/zup/op/events/application/KafkaEventManager.kt

```diff
@@ -5,31 +5,29 @@ import br.com.zup.op.events.domain.EventRepository
 import br.com.zup.op.events.domain.ReasonEntity
 import br.com.zup.op.events.domain.ReasonRepository
 import br.com.zup.op.events.domain.TopicEntiy
-import br.com.zup.op.events.domain.TopicRepository
 import br.com.zup.op.events.interfaces.model.RepublishEventRequest
 import br.com.zup.op.events.interfaces.model.RepublishEventResponse
 import br.com.zup.op.events.producer.KafkaProducerConfig
 import com.fasterxml.jackson.databind.ObjectMapper
-
+import org.apache.kafka.clients.consumer.Consumer
 import org.springframework.stereotype.Service
-import kotlin.collections.ArrayList
+
 
 @Service
 class KafkaEventManager(
         private val eventRepository: EventRepository,
         private val reasonRepository: ReasonRepository,
-        private val topicRepository: TopicRepository,
-        private val kafkaProducerConfig: KafkaProducerConfig) : EventManager {
+        private val kafkaProducerConfig: KafkaProducerConfig,
```

Comment:
Remove the paramenter "kafkaProducerConfig".

Reply:
The parameter will be removed after @Willcd  commit a code publish event